spp_change_request/wizard/confirm_user_assignment.py

```diff
@@ -10,6 +11,15 @@ class ConfirmUserAssignmentWiz(models.TransientModel):
     _name = "spp.change.request.user.assign.wizard"
     _description = "Change Request User Assignment Wizard"
 
+    GROUP_NAMES = [
+        "Change Request Agent",
+        "Change Request Validator",
+        "Change Request Applicator",
+        "Change Request Administrator",
+        "Change Request Validator HQ",
+        "Change Request Validator Local",
+    ]
```

Comment:
It would be better to use the XML ID of these groups so that changes in the group names will not require editing this in the future.

Reply:
HQ and Local is too specific to the implemented project to be in OpenSPP.